compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -589,12 +589,20 @@ convType env reexported =
             args <- mapM (convType env reexported) lfArgs
             pure $ HsParTy noExt (noLoc $ foldl (HsAppTy noExt . noLoc) tyvar (map noLoc args))
 
+        LF.TApp (LF.TCon LF.Qualified {..}) (LF.TStruct fields)
+            | qualModule == LF.ModuleName ["DA", "Internal", "PromotedText"]
```

Comment:
Given that we have this in a stable package, why nto check the package id?

Reply:
Good idea :)

Reply:
Done!